tests/ResourceGeneratorTest.php

```diff
@@ -152,6 +152,27 @@ public function it_accepts_attributes_which_are_saved_to_the_model()
     }
 
     /** @test */
+    public function it_accepts_float_attributes_which_are_saved_to_the_model()
+    {
+        $model = 'Louse';
+
+        // When I create a new model called 'Animal'.
+        $this->runArtisanCommand(ResourceMakeCommand::class, [
+            'name' => $model,
+            'attributes' => 'eyes:float',
```

Comment:
The example attribute of `eyes` doesn't really fit with a float data type. Can you change this to `weight`, or something that could reasonably be a float value?